universal-login-sdk/src/core/services/WalletService.ts

```diff
@@ -149,18 +149,18 @@ export class WalletService {
     };
     return new Promise((resolve, reject) => {
       const setWallet = this.setWallet.bind(this);
-      const subscription = this.sdk.subscribe('KeyAdded', filter, () => {
+      const unsubscribe = this.sdk.subscribe('KeyAdded', filter, () => {
         setWallet(connectingWallet);
-        subscription.remove();
+        unsubscribe();
         resolve();
       });
-      connectingWallet.setSubscription(subscription);
+      connectingWallet.unsubscribe = unsubscribe;
     });
   }
 
   async cancelWaitForConnection() {
     if (this.state.kind === 'Deployed') return;
-    this.getConnectingWallet().subscription && this.getConnectingWallet().subscription!.remove();
+    this.getConnectingWallet().unsubscribe && this.getConnectingWallet().unsubscribe!();
```

Comment:
We don't need  `this.getConnectingWallet().unsubscribe && ...` `this.getConnectingWallet().unsubscribe!()` is enough.